### DISCUSSION TITLE
issue with placing boundaries for the scenes

### DISCUSSION THREAD

**Discussion by mozillaroom** [mozilla/hubs#6228] [Spoke Questions]
i would like to  place boundaries for the avatar to not to cross the scene.  i am new to Mozilla Hubs, and i dont have any idea on how i can achieve it. Can anyone help me with this issue.

**Comment by matthewbcool** [comment]
Hi @mozillaroom 

The navigation mesh dictates avatar's movement. You can read more about it here: https://hubs.mozilla.com/labs/what-is-a-nav-mesh/

The link above also contains a more advanced tutorial about customizing the nav mesh.
